miio/airqualitymonitor.py

```diff
@@ -92,16 +102,46 @@ def sensor_state(self) -> str:
         """Sensor state."""
         return self.data["sensor_state"]
 
+    @property
+    def co2(self) -> int:
+        return self.data["co2"]
```

Comment:
Please add docstrings to the new properties. Also, in case these methods do not work on the other devices, maybe consider rising an exception (or maybe even having a separate status class inheriting from the common one (if there are common elements)?